unity-renderer/Assets/DCLPlugins/ECS7/ECSComponents/GltfContainer/GltfContainerCollidersHandler.cs

```diff
@@ -0,0 +1,180 @@
+using DCL.Configuration;
+using DCL.Helpers;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.ECSComponents
+{
+    public class GltfContainerCollidersHandler
+    {
+        private readonly List<Collider> invisibleMeshesColliders = new List<Collider>(10); // colliders from `_collider`
+        private readonly List<Collider> visibleMeshesColliders = new List<Collider>(10); // colliders from renderers
+        private readonly List<Collider> visibleMeshesInvalidColliders = new List<Collider>(10); // colliders created from sdk6 but not valid for sdk7
```

Comment:
How are we gonna end up receiving colliders created in the SDK6 here?

Reply:
`GltfShape` creates colliders inside the instanced gltf hierarchy. 
Those colliders are not destroyed when `GlftShape` is removed,
but the gltf ends up in the pool of the gltf promise keeper library, with those colliders created.
Which I think it's OK, we just need to handle that scenario.